packages/plans-grid-next/src/components/features-grid/plan-features-list.tsx

```diff
@@ -59,13 +72,45 @@ const PlanFeaturesList = ( {
 
 	return plansWithFeatures.map(
 		( { planSlug, features: { wpcomFeatures, jetpackFeatures } }, mapIndex ) => {
-			const filteredWpcomFeatures = wpcomFeatures.filter(
-				( feature ) => featureGroup?.getFeatures().includes( feature.getSlug() )
-			);
+			// This is temporary. Ideally all feature-groups will include the included features.
+			const filteredWpcomFeatures =
+				featureGroup?.slug === FEATURE_GROUP_ALL_FEATURES
+					? wpcomFeatures
+					: wpcomFeatures.filter(
+							( feature ) => featureGroup?.getFeatures().includes( feature.getSlug() )
+					  );
 
-			if ( featureGroup && ! filteredWpcomFeatures.length ) {
+			/**
+			 * 1. Storage group is still it's own thing, with no actual features associated. It will join the rest in a follow-up.
+			 */
+			if ( FEATURE_GROUP_STORAGE === featureGroup?.slug ) {
+				return (
+					<PlanDivOrTdContainer
+						key={ `${ planSlug }-${ mapIndex }` }
+						isTableCell={ options?.isTableCell }
+						className="plan-features-2023-grid__table-item"
+					>
+						<PlanFeaturesItem>
+							<h2 className="plans-grid-next-features-grid__feature-group-title">
+								{ featureGroup?.getTitle() }
+							</h2>
+						</PlanFeaturesItem>
+						<PlanStorageOptions
```

Comment:
Let's also rename this to `PlanStorageOption` - although in a follow-up. Ideally, we would render the associated feature (when we introduce the slugs to the storage group) or the selector as a separate thing.